src/baselaplace/predicting.jl

```diff
@@ -73,36 +104,68 @@ predict(la, hcat(x...))
 ```
 """
 function predict(
```

Comment:
This looks fine for now, although with all these conditionals we introduce a lot of computational overhead. But that's one for another PR

Reply:
Yeh let's not worry about this for now I would say :)